pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -260,15 +260,46 @@ def test_to_datetime_tz_pytz(self):
                                  dtype='datetime64[ns, UTC]', freq=None)
         tm.assert_index_equal(result, expected)
 
-    def test_to_datetime_utc_is_true(self):
-        # See gh-11934
-        start = pd.Timestamp('2014-01-01', tz='utc')
-        end = pd.Timestamp('2014-01-03', tz='utc')
-        date_range = pd.bdate_range(start, end)
-
-        result = pd.to_datetime(date_range, utc=True)
-        expected = pd.DatetimeIndex(data=date_range)
-        tm.assert_index_equal(result, expected)
+    @pytest.mark.parametrize("init_constructor, end_constructor, test_method",
+                             [(Index, DatetimeIndex, tm.assert_index_equal),
+                              (list, DatetimeIndex, tm.assert_index_equal),
+                              (np.array, DatetimeIndex, tm.assert_index_equal),
+                              (Series, Series, tm.assert_series_equal)])
+    def test_to_datetime_utc_true_with_constructors(self,
+                                                    init_constructor,
+                                                    end_constructor,
+                                                    test_method):
+        # See gh-11934 & gh-6415
+        data = ['20100102 121314', '20100102 121315']
+        expected_data = [pd.Timestamp('2010-01-02 12:13:14', tz='utc'),
+                         pd.Timestamp('2010-01-02 12:13:15', tz='utc')]
+
+        result = pd.to_datetime(init_constructor(data),
+                                format='%Y%m%d %H%M%S',
+                                utc=True)
+        expected = end_constructor(expected_data)
+        test_method(result, expected)
+
+        # Test scalar case as well
+        for scalar, expected in zip(data, expected_data):
+            result = pd.to_datetime(scalar, format='%Y%m%d %H%M%S', utc=True)
+            assert result == expected
+
+    def test_to_datetime_utc_true_with_series_single_value(self):
+        # GH 15760 UTC=True with Series
+        ts = 1.5e18
+        result = pd.to_datetime(pd.Series([ts]), utc=True)
+        expected = pd.Series([pd.Timestamp(ts, tz='utc')])
+        tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize("dtype", [None, 'datetime64[ns]'])
+    def test_to_datetime_utc_true_with_naive_series(self, dtype):
```

Comment:
you're actually testing the aware case no (instead of naive? as the naive is already included in test above)

Reply:
I don't exactly remember why I added this test, but I think I was testing `naive` dtypes (i.e. the string is tz-aware but the Series is constructed with `object` and  `datetime64[ns]` dtypes). However, the tz-aware string gets localized correctly with `datetime64[ns]` anyways before `to_datetime`...

I'll parametrize this test to test naive and aware date strings with naive dtype construction and clarify the name of this test.

Reply:
ah, I missed the tz part in the string. So it is about a 'tz-aware string' then?

Reply:
The test was more about constructing a Series with a non tz-aware dtype (`object` or `datetime[ns]`), and then localizing it to UTC with `to_datetime`. I just so happened to use a tz-aware string.